ister.py

```diff
@@ -1422,6 +1478,12 @@ def install_os(args, template):
         copy_os(args, template, target_dir)
         add_users(template, target_dir)
         set_hostname(template, target_dir)
+        if template.get("MirrorURL"):
+            cmd_mirror_url =  template.get("MirrorURL")
```

Comment:
Minor nit, extra space after =. What do you think about just passing template instead of the specific thing from the template? All the other calls around this one do that so it would be more uniform.

Reply:
No problem...now that I am not writing args to any destination - I don't need the extra logic, and can make it consistent like the other calls